plasmapy/physics/transport/tests/test_collisions.py

```diff
@@ -38,6 +38,12 @@ def setup_class(self):
         self.gms6 = 3.635342040477818
         self.gms6_negative = 0.030720859361047514
 
+    def test_symmetry(self):
+        lnLambda = Coulomb_logarithm(self.temperature1, self.density2, self.particles)
+        lnLambdaRev = Coulomb_logarithm(self.temperature1, self.density2, self.particles[::-1])
+        assert lnLambda == lnLambdaRev
```

Comment:
add error string for test failure

Reply:
I honestly think we can skip that! Here, let me show you what happens if I mess up this test by taking the other temperature times 20. This is part of my pytest output:
```python
========================================= FAILURES ==========================================
___________________________ Test_Coulomb_logarithm.test_symmetry ____________________________

self = <plasmapy.physics.transport.tests.test_collisions.Test_Coulomb_logarithm object at 0x7f7978f4d198>

    def test_symmetry(self):
        lnLambda = Coulomb_logarithm(self.temperature1, self.density2, self.particles)
        lnLambdaRev = Coulomb_logarithm(self.temperature1*20, self.density2, self.particles[::-1])
>       assert lnLambda == lnLambdaRev
E       assert 0.03193239302349768 == 3.8745137152918026

plasmapy/physics/transport/tests/test_collisions.py:44: AssertionError
```

That seems clean and understandable enough to me! Sure, the `self = object at hex address in memory` part is unhelpful, but the test is simple enough to see where you need to start digging without further prompting.

Reply:
Fair point, here it is not so ambiguous what is going on. It's just that in general our variable names aren't necessarily clear as to what is the expected value (and why) and what is the value obtained from the function.

Reply:
That just means we have to write better tests!

Reply:
You mean like clearly documenting what is going on in the form of an error string? :grin:

Reply:
I think well written and well named tests can actually be self.documenting. :smiley: